### PR TITLE
Release v4.2.0

### PR DESCRIPTION
**Pull request by gregyu**

### CODE REVIEW

File: knockoff/utilities/ioc.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    container.wire(modules=[sys.modules[__name__]])
+    return container
+
+
+def _validate_container_class(cls, package_name):
+    if not inspect.isclass(cls) or not issubclass(cls, containers.DeclarativeContainer):
+        raise TypeError(f"{package_name} resolves to "
+                         f"{cls} instead of "
+                         f"a subclass of dependency_injector"
+                         f".containers.DeclarativeContainer")
```
Comment (abdulhuq811) [inline knockoff/utilities/ioc.py:32]: Needs a new line here

File: knockoff/utilities/ioc.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2021-present, Nike, Inc.
```
Comment (abdulhuq811) [inline knockoff/utilities/ioc.py:1]: What does ioc stand for?

File: knockoff/utilities/ioc.py (hunk 1 above)
Comment (gregyu) [reply knockoff/utilities/ioc.py]: fixed

File: knockoff/utilities/ioc.py (hunk 2 above)
Comment (gregyu) [reply knockoff/utilities/ioc.py:1]: inversion of control which we get with dependency injection

**Review by abdulhuq811 — APPROVED** [review]

**Pull request merged**